### ISSUE TITLE
Labels information on BoundingBoxes3D.

### ISSUE THREAD

**Issue by Ardorax** [Visual-Behavior/aloception-oss#253]
The BoundingBoxes3D is always the same color and there isn't any information about the Label.
```py
import numpy as np
import torch
from aloscene import Frame, Labels, BoundingBoxes3D, CameraExtrinsic, CameraIntrinsic, BoundingBoxes2D

frame = Frame(np.zeros((3, 500, 500)), normalization="01")
frame.append_cam_intrinsic(CameraIntrinsic(focal_length=721.0, plane_size=frame.HW))
frame.append_cam_extrinsic(CameraExtrinsic(torch.eye(4, dtype=torch.float32)))
label = Labels([1], names=["name"])

box3d = BoundingBoxes3D([[10, 10, 400, 80, 46, 18, 1]], labels=label)
frame.append_boxes3d(box3d)

box2d = BoundingBoxes2D([[200, 200, 200, 300]], boxes_format="xcyc", absolute=True, frame_size=(500, 500), labels=label)
frame.append_boxes2d(box2d)

# No data about labels is displayed next to boxes3d.
frame.get_view().render()
```

**Comment by Ardorax** [comment]
No the bug is still here.

**Comment by anhtu293** [comment]
I can get different color and information for 3d boxe on my side with the given code.

**Comment by Ardorax** [comment]
Are you sure you are on the alobugsdays branch ? Can you send a screenshot (with the output of `git log -7` in your terminal)  ?

**Comment by anhtu293** [comment]
![image](https://user-images.githubusercontent.com/35274690/202464375-2355c837-8bf4-4f99-acb4-374324324f63.png)

**Comment by Ardorax** [comment]
The 3d bow will always be green and blue so the bug is still here. More, you can see the number next to the 3d box but not the 3d box.
